### ISSUE TITLE
v1.0.2 Client crash on startup

### ISSUE THREAD

**Issue by briansemrau** [RedstoneParadox/NiceToHave#5]
**Nice to Have version:** 1.0.2

**Fabric Loader Version:** 0.4.8 build 158

**Fabric API Version:** 0.3.0 build 207

**Fabric Language Kotlin Version:** 1.3.40 build 1

**Name and version of any other mods present:** A lot to list, included in log

**What actually happened:** Client crashes on startup

**Steps to reproduce:** Try to start minecraft client

Crash log: https://paste.ee/p/wa8c2
```
java.lang.NoClassDefFoundError: net/minecraft/entity/EntityCategory
at net.redstoneparadox.nicetohave.entity.EntityTypes.<clinit>(EntityTypes.kt:14)
at net.redstoneparadox.nicetohave.NiceToHave.onInitialize(NiceToHave.kt:22)
```

**Comment by RedstoneParadox** [comment]
Can I also ask what mappings version you were using?

**Comment by RedstoneParadox** [comment]
Nevermind, I got it working and it should be fixed in 1.0.3
